ports/raspberrypi/bindings/rp2pio/StateMachine.c

```diff
@@ -24,9 +24,13 @@
 //| import memorymap
 //|
 //| FifoType = Literal["auto", "txrx", "tx", "rx", "txput", "txget", "putget"]
+//| """A type representing of the strings ``"auto"``, ``"txrx"``, ``"tx"``, ``"rx"``, ``"txput"``, txget" or ``"putget"``. These values are supported on RP2350. For type-checking only, not actually defined in CircuitPython."""
```

Comment:
Tiny typo
```suggestion
//| """A type representing of the strings ``"auto"``, ``"txrx"``, ``"tx"``, ``"rx"``, ``"txput"``, ``"txget"`` or ``"putget"``. These values are supported on RP2350. For type-checking only, not actually defined in CircuitPython."""
```

Reply:
Also should be `one of the strings`, not `of the strings`.

Reply:
thanks, I totally overlooked it including in the rendered docs. fixed now.